server/src/unifyfs_inode.c

```diff
@@ -530,23 +540,25 @@ int unifyfs_inode_resolve_extent_chunks(unsigned int n_extents,
 
         chunk_read_req_t* pos = chunks;
         for (i = 0; i < n_extents; i++) {
+            chunk_read_req_t* ext_chunks = resolved[i];
             for (j = 0; j < n_resolved[i]; j++) {
-                *pos = resolved[i][j];
+                /* MJB_TODO("remove this debug print") */
+                debug_print_chunk_read_req(ext_chunks + j);
+                *pos = ext_chunks[j];
```

Comment:
This reminder intended for now? Or sometime in the future?

Reply:
Intended for now. Will remove it.

Reply:
fixed this one, and found another MJB_TODO reminder that I took care of as well